third_party/move/move-prover/src/cli.rs

```diff
@@ -808,11 +808,18 @@ impl Options {
                     "../../../../aptos-move/framework/src/aptos-natives.bpl"
                 )
                 .to_vec(),
-                module_instance_names: vec![(
-                    "0x1::object".to_string(),
-                    "object_instances".to_string(),
-                    true,
-                )],
+                module_instance_names: vec![
```

Comment:
Seems like `module_instances` can be factored out? It is duplicated across multiple files causing more changes

Reply:
done